### PR TITLE
(Refactor): Typo Fix in the repeated properties of sample engine spec

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <uditgaurav@gmail.com>

This pr is for typo fixing in the sample chaos engine snippet - Removing the repeated properties from the ChaosEngine spec

### CODE REVIEW

File: docs/cpu-hog.md [hunk 1]
```diff
@@ -113,16 +113,14 @@ metadata:
   name: nginx-chaos
   namespace: default
 spec:
-  chaosType: 'infra'  # It can be app/infra
-  auxiliaryAppInfo: "ns1:name=percona,ns2:run=nginx"
+# It can be app/infra
```
Comment (chandankumar4) [inline docs/cpu-hog.md]:
```suggestion
  # It can be app/infra
```

Comment (chandankumar4) [inline docs/cpu-hog.md]: align the comment with chaosType word and for below also

Comment (uditgaurav) [reply docs/cpu-hog.md]: done

**Review by ispeakc0de — APPROVED** [review]
/lgtm

**Review by ksatchit — APPROVED** [review]

**Pull request merged**